lightning_server.cc

```diff
@@ -1,15 +1,13 @@
 #include "config_parser.h"
+#include "request_handlers.h"
```

Comment:
Addressed as part of https://github.com/UCLA-CS130/Mr.-Robot-et-al./pull/15/commits/83b15ef868d5dc7f6c0b4da4df3f155f05d6af12